src/components/GraphNode/GraphNode.js

```diff
@@ -165,6 +172,7 @@ GraphNode.defaultProps = {
   cursorType: 'pointer',
   renderPrependedInfo: noop,
   renderAppendedInfo: noop,
+  graphNodeRef: noop,
```

Comment:
should it have a default value? `undefined` is what react usually expects if you're not doing anything w/ the ref.